### DISCUSSION TITLE
Why type Prism.Ioc.IContainerRegistry not found ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2862] [Uno Platform] [answered]
For all details please see the source in repository : https://github.com/sigmarsson/UnoApp5

<img width="593" alt="image" src="https://user-images.githubusercontent.com/43340959/235327653-55096382-2a74-4277-84a3-19f9602d8dd4.png">

**Comment by dansiegel** [comment, marked as answer]
You have no reference to Prism.Core

**Reply by sigmarsson** [reply]
but I set the reference, that namespace can be found in assembly ```Prism.dll``` , which is referenced. And furthermore the type ```PrismApplication``` is found but the interface ```IContainerRegistry``` not. 

Here are some build context : 

```Files\dotnet\packs\Microsoft.NETCore.App.Ref\7.0.5\ref\net7.0\netstandard.dll".
2>                            Reference found at search path location "{RawFileName}".
2>                            This reference is not "CopyLocal" because at least one source item had "Private" set to "false" and no source items had "Private" set to "true".
2>                        Dependency "Mono.Android, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065".
2>                        C:\Users\weath\source\repos\UnoApp5\UnoApp5\UnoApp5\Prism\PrismApp.cs(24,47,24,65): error CS0246: The type or namespace name 'IContainerRegistry' could not be found (are you missing a using directive or an assembly reference?)
2>                      Done executing task "DetectInvalidResourceOutputFilenamesTask_07f22ce1031ceddcbe813390bd69b6f182b04501".
2>                      C:\Users\weath\source\repos\UnoApp5\UnoApp5\UnoApp5\Prism\PrismApp1.cs(17,32,17,38): error CS0108: 'PrismApp.Window' hides inherited member 'UIApplicationDelegate.Window'. Use the new keyword if hiding was intended.
2>                      Using "ResizetizeImages_07f22ce1031ceddcbe813390bd69b6f182b04501" task from assembly "C:\Users\weath\.nuget\packages\uno.resizetizer\1.0.2\build\Uno.Resizetizer_07f22ce1031ceddcbe813390bd69b6f182b04501.dll".
2>                      C:\Users\weath\source\repos\UnoApp5\UnoApp5\UnoApp5\Prism\App.xaml.cs(8,33,8,41): error CS0534: 'PrismApp' does not implement inherited abstract member 'PrismApplicationBase.RegisterTypes(IContainerRegistry)'```

**Reply by sigmarsson** [reply]
Strange that - for example - ```IContainerRegistryExtensions``` is available : 

![image](https://user-images.githubusercontent.com/43340959/235343636-560626f1-fd16-44a5-8e23-f3842aa9f2b7.png)

**Reply by sigmarsson** [reply]
<img width="661" alt="image" src="https://user-images.githubusercontent.com/43340959/235344165-8c9acc94-d221-43df-9fbd-0fa2e5522f44.png">

**Reply by sigmarsson** [reply]
@dansiegel I apologies. I was not aaware the new Uno template has phased out shared project and hence the new (common project) references too the assemblies.

**Reply by dansiegel** [reply]
What you're working on right now while it should work is not fully supported as it is not publicly released bits. As I get time I will get this out.
